Unreal/CarlaUnreal/Plugins/Carla/Source/Carla/Settings/CarlaSettingsDelegate.cpp

```diff
@@ -422,20 +422,20 @@ void UCarlaSettingsDelegate::SetAllLights(
     const bool cast_shadows,
     const bool hide_non_directional) const
 {
-  if (!world || !IsValid(world) || world->IsPendingKill())
+  if (!world || !IsValid(world))
```

Comment:
```suggestion
  if (!IsValid(world))
```
IsValid() is already doing the nullcheck.